### PR TITLE
66220014825374135: New upload image editor flow

### PR DESCRIPTION
**Pull request by cristobalchao**
## :pushpin: References

- **Wonder issue:** https://app.wonderverse.xyz/organization/wonderverse/boards?task=67914566093243214
- **Related pull-requests:**

## :blue_book: Description

Implements the new profile and cover image cropping flow. As well, as making the "Profile Header" responsive.

## :movie_camera: Screenshot or Video

image

https://user-images.githubusercontent.com/2279571/193346239-fc668311-0bb9-4e3b-843b-727e8b1d2827.mov

## :cake: Checklist:

- [x] I self-reviewed my changes
- [x] My changes follow the style guide: https://creative-earth-33e.notion.site/TT-Wonder-Style-guide-4702a45133374148953bfcaf584120b7
- [x] I tested my changes in Chrome

### CODE REVIEW

**Comment by terryli0095** [comment]
there's still no delete button on the page?

**Comment by terryli0095** [comment]
Screen Shot 2022-11-14 at 4 41 48 PM
there should be delete and replace buttons next to profile pic

File: wondrous-app/components/Settings/imageUpload.tsx [hunk 1]
```diff
@@ -145,6 +178,16 @@ export function ImageUpload(props) {
             onChange={handleNewFileUpload}
           />
         </ImageUploadBlockInputWrapper>
+        {imageType === 'ICON_IMAGE' && imageInputField.current?.value ? (
```
Comment (terryli0095) [inline wondrous-app/components/Settings/imageUpload.tsx:181]: i think it's because imageInputField.current?.value is undefined

Comment (terryli0095) [reply wondrous-app/components/Settings/imageUpload.tsx:181]: unless you actually change the image once

File: wondrous-app/components/Settings/imageUpload.tsx [hunk 2]
```diff
@@ -44,6 +46,8 @@ export function ImageUpload(props) {
   const [imageToEdit, setImageToEdit] = useState(null);
   const [editedImage, setEditedImage] = useState(null);
 
+  console.log('ImageType', imageType);
```
Comment (terryli0095) [inline wondrous-app/components/Settings/imageUpload.tsx:49]: remove log

Comment (ivandonyk) [reply wondrous-app/components/Settings/imageUpload.tsx:49]: Oh, my bad, I forgot to remove it.

**Pull request merged**